src/nvim/lua/treesitter.c

```diff
@@ -64,6 +64,7 @@ static struct luaL_Reg tree_meta[] = {
   { "__tostring", tree_tostring },
   { "root", tree_root },
   { "edit", tree_edit },
+  { "included_ranges", tree_get_ranges },
```

Comment:
nit: I see that `included_ranges` already exists on parser, otherwise I wonder why not name this `get_ranges` ? Helps with discovery since we have `get_range` (singular).

Reply:
This is quite low level anyway. When in doubt, we just match upstream.